src/me/components/Resources.tsx

```diff
@@ -64,6 +69,9 @@ const ResourceLists: FC = () => {
           </Panel>
         </>
       )}
+      {isFlagEnabled('uiUnificationFlag') &&
+        isFlagEnabled('paygCheckoutCredit') &&
+        paygCreditEnabled && <UsagePanel />}
```

Comment:
something about this makes me feel weird but 🤷🏽

Reply:
me too!

Reply:
Like, it feels weird to me that this particular component may or may not have access to context based on the conditional wrapper above. We should probably default `paygCreditEnabled` to false, then check to see if we're in a place where `uiUnification` is on, then set the value based on that, but I'm not sure if that's kosher in terms of react hooks being called within `ifs`